test/TestAssets/NewtonSoftDependency/NewtonSoftDependency.csproj

```diff
@@ -26,7 +26,7 @@
   <ItemGroup>
     <PackageReference Include="MSTest.TestAdapter" Version="$(MSTestAdapterVersion)" />
     <PackageReference Include="MSTest.TestFramework" Version="$(MSTestAdapterVersion)" />
-    <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
```

Comment:
Could you use the msbuild variable here?

Reply:
I think the intent here was to use a version that is newer than what we have in TP. But currently there is no such stable version. Added comment.